src/utils/messages.ts

```diff
@@ -82,7 +82,7 @@ export function getBotWelcomeMessages(
 }
 
 export function isFormMessage(message: EveryMessage) {
```

Comment:
tip. by using a type predicate, you don't need to perform type assertion `as Form[]`.

```suggestion
export function isFormMessage(message: EveryMessage): message is EveryMessage & { extendedMessagePayload: { forms: Form[] }} {
```